### ISSUE TITLE
repo-actions links do not appear if there is no TOC

### ISSUE THREAD

**Issue by maelle** [quarto-dev/quarto-cli#2142]
### Bug description

I created a book project using RStudio and only added the `repo-url` and `repo-actions`

In `_quarto.yml` I have

```yaml
book:
  title: "sidebarbug"
  author: "Jane Doe"
  date: "8/26/2022"
  chapters:
    - index.qmd
    - intro.qmd
    - summary.qmd
    - references.qmd
  repo-url: https://github.com/maelle/sidebarbug/
  repo-actions: [edit, issue]
```

Source: https://github.com/maelle/sidebarbug

Now on the page where there is a TOC (because there are headers) the repo action links are present. https://maelle.github.io/sidebarbug/
But on the pages where there is no TOC (which is expected as there are no headers) the repo action links are not present (which I was not expecting). https://maelle.github.io/sidebarbug/intro.html

I don't think this is relevant but just in case, I am using 
RStudio 2022.07.0+548 "Spotted Wakerobin" Release (34ea3031089fa4e38738a9256d6fa6d70629c822, 2022-07-06) for Ubuntu Bionic
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36
 Ubuntu 20.04.4 LTS

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by elgabbas** [comment]
I recently noticed the exact issue using v1.3.24.

**Comment by dragonstyle** [comment]
This should be fixed in our next pre-release build. You have a couple of choices:

1) Force the actions to appear in the sidebar even when there is no TOC by adding `repo-actions: true` to your `qmd` like so:

```markdown
---
repo-actions: true
---

This is a document with no header or TOC
```

2) Starting in Quarto 1.3, we will attempt to place the tools in the TOC or sidebar if we can, and if none of those are present, we will place them in the footer of the page.

Thanks for reporting this!

**Comment by maelle** [comment]
Thanks for this fix and the one for #4404! :rocket: :pray:
